dev

qqalex93 opened the pull request.

Commit:
Внесены изменения по домашней раоте 11 спринта. Обновлены файлы верстки. Добвлены классы

Commit:
Внесены изменения в код для домашнего задания №12
добавлены векторные изображения для темной и светлой тем. Добавлена реализация поиска музыки через API itunes

IliaAnK commented on app/src/main/java/com/practicum/playlistmaker/SearchActivity.kt:
```diff
@@ -2,29 +2,72 @@ package com.practicum.playlistmaker
+            .baseUrl(baseUrlITunes)
+            .addConverterFactory(GsonConverterFactory.create())
+            .build()
+        trackApi = retrofit.create(TrackApi::class.java)
```
Рекомендация:
Создание всего ретрофит клиента можно вынести в отдельный класс/файл

qqalex93 replied:
Не совсем понял рекомендацию про баг с историей, ибо по 11 спринту этого еще не было

IliaAnK replied:
Прошу прощения, упоминание истории случайно попала туда. На данном этапе её(историю) можно просто проигнорировать

Commit:
Внесены изменения в код для домашнего задания №11
исправлено замечание ревьюера. Учтены рекомендации. Слегка оптимизирован файл верстки
